### PR TITLE
Implement ChannelFileSummary viewset

### PR DESCRIPTION
**Pull request by jonboiser**
Implements #1492

`http://localhost:8000/api/content/8b4d3e6d3d4842ba8ea658335b5dd252/filesummary/`

For example, the "small testing channel" Returns
```

    {
        "total_files": 74,
        "total_file_size": 35592558
    }

```

### CODE REVIEW

File: kolibri/content/api.py [hunk 1]
```diff
@@ -231,3 +231,13 @@ class FileViewset(viewsets.ModelViewSet):
...
+
+class ChannelFileSummarySerializer(viewsets.ModelViewSet):
+    serializer_class = serializers.ChannelFileSummarySerializer
+
+    def get_queryset(self):
+        aggs = models.File.objects.aggregate(
+            total_files = Count('pk'),
+            total_file_size = Sum('file_size')
+        )
+        return [aggs]
```
Comment (jonboiser) [inline kolibri/content/api.py]: I cannot simply return `aggs`. Need to wrap it in an array to make it work.

Comment (rtibbles) [reply kolibri/content/api.py]: @jonboiser probably best to do this on the serializer - the queryset object should be a queryset (or, as you discovered, at the very least an iterable, hence wrapping it in a list works).

Comment (rtibbles) [reply kolibri/content/api.py]: `from rest_framework.response import Response`

```
class ChannelFileSummaryViewSet(viewsets.ViewSet):
    def list(self, request):
        aggs = models.File.objects.aggregate(
            total_files = Count('pk'),
            total_file_size = Sum('file_size')
        )
        return Response(aggs)
```

**Comment by jonboiser** [comment]
@rtibbles Rewrote the ViewSet the way you suggested, and it works!
screen shot 2017-05-31 at 11 48 27 am

**Comment by jonboiser** [comment]
I'm going to close this and cherry pick it in to #1603

**Pull request closed without merging**